package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "redo-monolith",
-  "version": "1.0.1",
+  "version": "2.0.0",
```

Comment:
What happens with the previewsly version released as 2.0.0? Why does GH shows the change if master has 2.0.0? I'm confused (I trust it works for us but just to be sure)